viofs/svc/virtiofs.c

```diff
@@ -321,7 +321,7 @@ static VOID SetFileInfo(struct fuse_attr *attr, FSP_FSCTL_FILE_INFO *FileInfo)
 {
     FileInfo->FileAttributes = PosixUnixModeToAttributes(attr->mode);
     FileInfo->ReparseTag = 0;
-    FileInfo->AllocationSize = attr->blocks * attr->blksize;
+    FileInfo->AllocationSize = attr->blocks * 512;
```

Comment:
Shouldn't we query volume for this info?

Reply:
According to the man page (https://linux.die.net/man/2/stat), no: number of 512B blocks allocated.